File: comms/bulk-messaging/sender-pools/update-sender-pool/update-sender-pool.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\SenderPools\Requests\SenderPoolsUpdateRequest;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->senderPools->update(
    'comms_senderpool_01h9krwprkeee8fzqspvwy6nq8',
    new SenderPoolsUpdateRequest([
        'name' => 'Customer Support Senders - Tier 1',
        'tags' => [
            'tier' => '1',
        ],
    ]),
);

File: comms/bulk-messaging/sender-pools/update-sender-pool/update-sender-pool.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "TWILIO_ACCOUNT_SID",
  auth_token: "TWILIO_AUTH_TOKEN"
)

client.sender_pools.update(
  sender_pool_id: "comms_senderpool_01h9krwprkeee8fzqspvwy6nq8",
  name: "Customer Support Senders - Tier 1",
  tags: {
    tier: "1"
  }
)
